### PR TITLE
fix(Rich Text Editor): Removed editor id prop from parsing method

### PR DESCRIPTION
**Pull request by bojangles-m**
It is breaking the Clarify test with infinite rendering.
The Fondue beta 10 is working normally the 11 is breaking.

### CODE REVIEW

**Review by SamCreasey — APPROVED** [review]
@Walkeezy Seems like we might need an alternative solution to the infinite rendering caused by not supplying the id as it causes other problems, namely that the infinite loop is still visible within tests in Clarify but it's not 100% clear on why yet, only that it is caused by the parseRawValue function.

File: src/components/RichTextEditor/utils/parseRawValue.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const parseRawValue = (editorId: string, raw?: string): Value => {
     try {
         parsedValue = JSON.parse(raw);
     } catch {
-        const editor = createPlateEditor({ id: editorId, plugins: getEditorConfig() });
+        const editor = createPlateEditor({ plugins: getEditorConfig() });
```
Comment (Walkeezy) [inline src/components/RichTextEditor/utils/parseRawValue.ts:24]: Do the tests in clarify work when doing something like this?

```suggestion
        const editor = createPlateEditor({ id: editorId + '_parseRawValue', plugins: getEditorConfig() });
```

File: src/components/RichTextEditor/RichTextEditor.cy.tsx [hunk 2]
```diff
@@ -150,6 +156,13 @@ describe('RichTextEditor Component', () => {
         cy.get(RICH_TEXT_EDITOR).should('contain.text', TEXT);
     });
 
+    it('should be possible to pass and change value', () => {
```
Comment (Walkeezy) [inline src/components/RichTextEditor/RichTextEditor.cy.tsx:159]: The description of this test case seems missleading, as the actual test is just passing and not changing the value.

Comment (bojangles-m) [reply src/components/RichTextEditor/RichTextEditor.cy.tsx]: Updated

**Review by Walkeezy — APPROVED** [review]

**Pull request merged**